sycl/doc/extensions/proposed/sycl_ext_intel_fpga_kernel_interface_properties.asciidoc

```diff
@@ -218,6 +245,29 @@ inline constexpr pipelined_key::value_t<
 
  If the property parameter (N) is not specified, the default value is `-1`.
  Valid values for `N` are values greater than or equal to `-1`.
+
+|`fpga_cluster`
+|The `fpga_cluster` property adds a request, to the extent possible, for the
+ compiler to implement statically-scheduled clusters using the specified
+ clustering method. The following values are supported:
+
+ * `stall_enable`: Directs the compiler to prefer generating an enable signal to
+    freeze the cluster when the cluster is stalled whenever possible.
+
+ * `stall_free`: Directs the compiler to prefer using an exit FIFO to hold
+    output data from clusters when the cluster is stalled whenever possible.
+
+ *Note*: Not all clusters can be implemented with the provided preference.
+
+ If the `fpga_cluster` property is not specified, the default behavior is
+ equivalent to one of the values listed above, but the choice is implementation
+ defined.
```

Comment:
We definitely shouldn't say this.  If the property is not specified we want to reserve the right to decide on a cluster-by-cluster basis whereas this text implies that we have to pick one type wherever possible and document that choice in our docs.  I'd remove this sentence entirely.